examples/bidder/ad_selector.hpp

```diff
@@ -112,16 +122,23 @@ class AdSelector {
                 return AdPtr();
             }
             const std::vector<Ad>::const_iterator result = std::max_element(ads.cbegin(), ads.cend(), [](const Ad &first, const Ad &second) -> bool {
-                return first.max_bid_micros < second.max_bid_micros;
+                return first.auth_bid_micros && second.auth_bid_micros ? 
+                       first.auth_bid_micros <second.auth_bid_micros : first.max_bid_micros < second.max_bid_micros;
```

Comment:
I think we have to insist on having auth_bid_micros , not sure how to do it yet. Basically if all selected Ads have no auth_bid_micros I think we have to produce NoBid response .